memstore/backfill_manager.go

```diff
@@ -150,6 +150,10 @@ func (r *BackfillManager) StartBackfill() ([]*UpsertBatch, int64, uint32) {
 	r.Lock()
 	defer r.Unlock()
 
+	utils.GetLogger().With("action", "Backfill", "table", r.TableName, "shard", r.Shard,
```

Comment:
I prefer to put all the logs in backfill.go so that we have a centralized place to put all backfill related logs

Reply:
this is log for the start of backfill which has all related info